index.html

```diff
@@ -2,9 +2,197 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
     <title>Title</title>
 </head>
+
+
 <body>
 
+<header>
+    <div class="outer-container">
+    <div class="inner-container">
+        <nav>
+            <ul>
+                <li><a href="#">Case studies</a></li>
+                <li><a href="#">Services</a></li>
+                <li><a href="#">Contact us</a></li>
+            </ul>
+        </nav>
+
+        <article class="header-opening">
+            <h1>Your website is an investment in the success of your business</h1>
+            <p class="opening-text">We are here to help you succeed</p>
+        <button type="button">let's work together</button>
+        </article>
+
+
+        <img class="squiggle" src="/assets/squiggle_line_stripe.svg" alt="squiggle-line">
```

Comment:
Op deze manier kan het plaatje niet getoond worden, omdat hij het plaatje niet kan vinden. Je was er bijna 😄 
```suggestion
        <img class="squiggle" src="assets/squiggle_line_stripe.svg" alt="squiggle-line">
```